### PR TITLE
getPlayers endpoint gives back a list of players that have done all actions

### PR DESCRIPTION
**Pull request by tomxhu**
I love underscore

### CODE REVIEW

File: api/entries.js [hunk 1]
```diff
@@ -184,19 +185,34 @@ module.exports = {
...
+        var playersThatDidActions = []
+
+        var promises = [];
+
+        actions.forEach(function(action){
+            promises.push(createFindPromise(action));
+        });
+
+        function createFindPromise(action) {
+            var deferred = Q.defer();
```
Comment (alexjohnson505) [inline api/entries.js:202]: YES! Good thinking to use a promise library....

**Comment by alexjohnson505** [comment]
:+1:

File: api/entries.js (hunk 1 above)
Comment (tomxhu) [inline api/entries.js:202]: Q is da best

**Comment by pragsanshi** [comment]
+1 merging

**Pull request merged**